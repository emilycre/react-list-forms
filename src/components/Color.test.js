import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color Component', () => {
  it('Renders a Color', () => {
    const wrapper = shallow(<Color name="Blue" hex="#0000FF" />);
    expect(wrapper).toMatchSnapshot();
  });
});
