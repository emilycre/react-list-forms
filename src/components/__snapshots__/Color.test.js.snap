// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Component Renders a Color 1`] = `
<dl>
  <dt>
    Name:
  </dt>
  <dd>
    Blue
     
    <div
      style={
        Object {
          "backgroundColor": "#0000FF",
          "display": "inline-block",
          "height": "0.8em",
          "width": "0.8em",
        }
      }
    />
  </dd>
  <dt>
    Hex:
  </dt>
  <dd>
    #0000FF
  </dd>
  <dt>
    RGB:
  </dt>
  <dd>
    <p>
      Red: 
      0
    </p>
    <p>
      Green: 
      0
    </p>
    <p>
      Blue: 
      255
    </p>
  </dd>
</dl>
`;
